.github/workflows/build.yml

```diff
@@ -0,0 +1,22 @@
+name: Basic Build
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '11', '17' ]
+    name: Basic Build (Java ${{ matrix.Java }})
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup java
+      uses: actions/setup-java@v3
+      with:
+        distribution: 'adopt'
+        java-version: ${{ matrix.java }}
+    - name: Test
+      run: ./gradlew ciBasicBuild
```

Comment:
Looking at those two tasks (`ciTest` and `ciBasicBuild`) it seems we're just wasting CI time. Building is done two times, so I guess an unified task is enough, it will just say "Build" and we know it also contains tests.

Instead of creating custom tasks you can just invoke:
```
gradlew build
```
And it will take care of building and testing what it can actually build and test.
Under Ubuntu it's going to be JVM, JS, and `linuxX64`.

Reply:
Thank you, I will run `./gradlew check -Ptarget.is.native=false.target.is.native=false`